src/main/java/uk/gov/companieshouse/web/pps/service/payment/PaymentService.java

```diff
@@ -9,6 +9,10 @@ public interface PaymentService {
     /**
      * Creates a payment session in order to pay for the Late Filing Penalty.
      */
-    String createPaymentSession(PayableLateFilingPenaltySession payableLateFilingPenaltySession, String companyNumber)
+    String createPaymentSession(
```

Comment:
Good practice where we spot this also, don't think we get any line length code smells in Sonar but something to keep in mind.